dataikuapi/dss/project.py

```diff
@@ -825,6 +825,54 @@ def new_job_definition_builder(self, job_type='NON_RECURSIVE_FORCED_BUILD'):
         warnings.warn("new_job_definition_builder is deprecated, please use new_job", DeprecationWarning)
         return JobDefinitionBuilder(self, job_type)
 
+    ########################################################
+    # Jupyter Notebooks
+    ########################################################
+    
+    def list_jupyter_notebooks(self, as_objects=True, active=False):
+        """
+        List the jupyter notebooks of a project.
+
+        :param bool as_objects: if True, return the jupyter notebooks as a :class:`dataikuapi.dss.notebook.DSSNotebook`
+                        notebook handles instead of raw JSON
+        :param bool active: if True, only return currently running jupyter notebooks.
+
+
+        :returns: The list of the notebooks - see as_objects for more information
+        :rtype: list
+        """
+        notebooks = self.client._perform_json("GET", "/projects/%s/jupyter-notebooks/" % self.project_key,
+                                         params={"active": active})
+        if as_objects:
+            return [DSSNotebook(self.client, notebook['projectKey'], notebook['name'], content=notebook) for notebook in notebooks]
```

Comment:
the call `jupyter-notebooks/` returns a `List<JupyterNotebookListItem>` whereas the DSSNotebook content should be a `NotebookSafeForWritingNew` especially that the DSSNotebook.save() send the content as the body and if you check the call PUT /{notebookName} we expect a NotebookSafeForWritingNew

Reply:
Ah yes actually it should be the state, I changed it in the last commit but it was correct before in fact